release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2476,6 +2476,65 @@ index a90ab14ff..f7f6f1a71 100644
+
+* Using NVIDIA-branded Mellanox NICs on Dell hardware, incoming packets that are larger than the preset F5 application receive buffer (currently set to 8K) arrive with an incorrect VLAN tag. This results in sporadic truncated packets being delivered by the NIC. (link:https://issues.redhat.com/browse/RHELPLAN-123058[*RHELPLAN-123058*])
+
+* Sometimes, during ZTP GitOps pipeline {sno} installations, the Operator Lifecycle Manager registry-server container fails to reach the `READY` state. When creating subscriptions with a new `CatalogSource`, the `CatalogSource` remains in a `TRANSIENT_FAILURE` state. (link:https://issues.redhat.com/browse/OCPBUGSM-44041[*OCPBUGSM-44041*])
+
+* Sometimes, a {sno} node configured with static IPv6 and deployed using the GitOps ZTP pipeline becomes unreachable during Day 2 operator configuration. The {product-title} cluster installation completes successfully, and the cluster is healthy. After reboot, the node becomes unreachable with the network interface down. (link:https://issues.redhat.com/browse/OCPBUGSM-46688[*OCPBUGSM-46688*])
+
+* During ZTP cluster deployments, the bare-metal host image provisioning fails with the following error: `The ETag supplied did not match the ETag required to change this resource`. To workaround this issue, attach the ISO image to the host using the Baseboard Management Controller GUI. (link:https://issues.redhat.com/browse/OCPBUGSM-46305[*OCPBUGSM-46305*])
+
+* During a master node replacement in a bare-metal cluster, the new master host gets stuck in a `Provisioning` state, but the node reports in to the cluster as `Ready`. (link:https://issues.redhat.com/browse/OCPBUGSM-45772[*OCPBUGSM-45772*])
+
+* At runtime, when host CPU cores are reserved separately for the cluster platform and deployed applications, some vDU pods use the reserved platform cores, and the {product-title} `etcd` datastore service uses isolated application cores. (link:https://issues.redhat.com/browse/OCPBUGSM-45078[*OCPBUGSM-45078*])
+
+* The log collector pod has higher CPU usage than the same pod in previous versions when you deploy a {sno} with the vRAN DU profile. (link:https://issues.redhat.com/browse/OCPBUGSM-46986[*OCPBUGSM-46986*])
+
+* When you apply a tuned override to a pod and delete the tuned pod to force a restart, the pod should restart and the system should run normally. Instead a `systemd` hang occurs and causes the system to stop responding. (link:https://issues.redhat.com/browse/RHELPLAN-131021[*RHELPLAN-131021*])
```

Comment:
Should say that a hang *may* occur and cause the system to stop responding (as discussed with Dwaine G that this issue was only seen one time).